File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int[] arr = new int[in.nextInt()];
        int tmp, i;
        for(i = 0; i < arr.length; i++){arr[i] = i+1;}
        int n = in.nextInt();
        for(i = 1; i <= n; i++){
            int a = in.nextInt()-1, b = in.nextInt()-1;
            tmp = arr[a];
            arr[a] = arr[b];
            arr[b] = tmp;
        }
        for(i = 0; i < arr.length; i++){
            System.out.printf("%d ", arr[i]);
        }
    }
}

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var arr = IntArray(sc.nextInt(),{0})
    for(i in 0..arr.size-1){
        arr[i] = i+1
    }
    for(i in 1..sc.nextInt()){
        val a = sc.nextInt()-1; val b = sc.nextInt()-1
        var temp = arr[a]
        arr[a] = arr[b]
        arr[b] = temp
    }
    arr.forEach { print("$it ") }
}